1-Basic/UserDetailsClient/UserDetailsClient.Droid/UserDetailsClient.Droid.csproj

```diff
@@ -35,7 +35,7 @@
     <WarningLevel>4</WarningLevel>
     <AndroidUseSharedRuntime>True</AndroidUseSharedRuntime>
     <AndroidLinkMode>None</AndroidLinkMode>
-    <EmbedAssembliesIntoApk>false</EmbedAssembliesIntoApk>
```

Comment:
This is needed in order for the the app to deploy to an android 11 device. Probably a bug somewhere in Xamarin.
I think this is what you ran into last week @jmprieur

Reply:
Can we link to information that explains why this is needed?

---
In reply to: [582408608](https://github.com/Azure-Samples/active-directory-xamarin-native-v2/pull/161#discussion_r582408608) [](ancestors = 582408608)